frontend/vue/main.ts

```diff
@@ -88,6 +89,7 @@ const components: any = {
   StatevectorBinaryEncoding,
   SyllabusLayout,
   SyllabusCreateLayout,
+  SyllabusForm,
```

Comment:
I think that the form is not used directly in a `.pug` template or chapter, so is not needed here.